packages/data-service/src/data-service.ts

```diff
@@ -1788,78 +1788,56 @@ export class DataServiceImpl implements DataService {
     return this._collection(ns, 'CRUD').bulkWrite(operations, options);
   }
 
-  currentOp(includeAll: boolean, callback: Callback<Document>): void {
+  async currentOp(includeAll: boolean): Promise<{ inprog: Document[] }> {
     const logop = this._startLogOp(
       mongoLogId(1_001_000_053),
       'Running currentOp'
     );
     const db = this._database('admin', 'META');
-
-    callbackify(db.command.bind(db))(
-      { currentOp: 1, $all: includeAll },
-      (error, result) => {
-        logop(error);
-        if (error) {
-          const logop = this._startLogOp(
-            mongoLogId(1_001_000_054),
-            'Searching $cmd.sys.inprog manually'
```

Comment:
This doesn't seems to work at all since server > 3.2 so I removed all this fallback logic as our driver version doesn't even support that version in the first place